src/Interface/InterfaceView/InterfaceView.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { KeyValue, Col, Row } from '@folio/stripes-components';
+
+import BoolToCheckbox from '../../Utils/BoolToCheckbox';
+
+import css from './InterfaceView.css';
+
+
+export const InterfaceView = ({ item = {}, thigh = false }) => { // eslint-disable-line import/prefer-default-export
```

Comment:
the same https://github.com/folio-org/ui-organizations/pull/57#discussion_r283075053

Reply:
Sorry, my fault, fixed.